server/app.py

```diff
@@ -326,25 +228,10 @@ def cancel_job(job_id):
         JobStatusResponse[status.CANCELED]
     """
     job = streaming_parent_task.AsyncResult(job_id)
-    input_job_id = job.info.get("input_job_id", None)
-    output_job_id = job.info.get("output_job_id", None)
-    if input_job_id:
-        input_job = process_file_streaming.AsyncResult(input_job_id)
-        input_job.revoke()
-    else:
-        logger.debug(
-            f"Input job id: {input_job_id} unavailable to cancel for parent job: {job_id}"
-        )
-    if output_job_id:
-        output_job = process_streaming_output.AsyncResult(output_job_id)
-        output_job.revoke()
-    else:
-        logger.debug(
-            f"output job id: {input_job_id} unavailable to cancel for parent job: {job_id}"
-        )
     job.revoke()
 
     # Delete Kafka topics
+    # TODO check this doesn't conflict with parent_job_error_handler
```

Comment:
has this been checked? Also does revoking the parent task actually kill the async tasks running within?

Reply:
if so it'd be a preexisting problem, haven't checked, it was a last minute thought I had before committing. We should check canceling async works though